portfolio/src/main/webapp/script.js

```diff
@@ -46,3 +46,12 @@ function addMoreQuotes() {
     document.getElementById('more-quotes-container').innerText = quote;
   });
 }
+
+/* Show favourite Pokemon to the page */
+function favouritePokemon() {
```

Comment:
nit: This naming isn't quite meaningful, shall we name it `showFavouritePokemon` intead?

Reply:
true, changed! thank you